apps/base-docs/docs/building-with-base/guides/account-abstraction/intro-to-account-abstraction.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Intro to Account Abstraction
+description: Learn what Account Abstraction it, how it works, and what problems it solves.
+keywords: [Privy, Account Abstraction, Base Paymaster, Account Kit, ERC-4337]
+hide_table_of_contents: false
+---
+
+[ERC-4337], also known as _Account Abstraction_, is a standard that allows smart contracts to initiate transactions, thus enabling any logic that users want to implement to be encoded into the smart contract wallet itself for execution on Ethereum.
+
+Account Abstraction has the potential to be a massive game-changer for onchain user experience and many believe it will play a key role in bringing the next billion users onchain.
+
+---
+
+## Objectives
+
+By the end of this guide you should be able to:
+
+- Explain how Account Abstraction can improve user experience for onchain apps
+- Describe the difference between contract accounts and EOAs, the limitations of contract accounts, and how EIP-4337 uses `UserOperation`s to mitigate these limitations
+- Outline how Account Abstraction works, and how users interact with smart contract wallets
+
+---
+
+## The Problem Account Abstraction Solves
+
+Onchain applications are difficult to use for many people, as they require a lengthy onboarding process for the average internet user who already onchain.
+
+For example,the preliminary steps a user needs to go through before they can use an onchain app are:
+
+1. Create a wallet wallet (Coinbase Wallet, Metamask, etc)
+1. Store the wallet mnemonic safely, without losing it or compromising it
```

Comment:
It's actually better to do it this way!  If you only use 1's it will automatically render as the numbers in order.  If you give it specific numbers and mess up later, it will render whatever numbers you put.